fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section4.java

```diff
@@ -0,0 +1,38 @@
+package ch.bbw.fabbwled.lands.book.book1;
+
+
+import ch.bbw.fabbwled.lands.book.SectionHandler;
+import ch.bbw.fabbwled.lands.book.SectionId;
+import ch.bbw.fabbwled.lands.book.SectionNode;
+import ch.bbw.fabbwled.lands.character.PlayerDto;
+import ch.bbw.fabbwled.lands.service.PlayerSession;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+public class Section4 implements SectionHandler {
+    private final PlayerSession playerSession;
+
+    @Override
+    public SectionId getId() {
+        return new SectionId(1, 4);
+    }
+
+
+
+    @Override
+    public SectionNode getBody(PlayerDto current) {
+        playerSession.update(x -> x.addShards(100).addPossession("trident  (COMBAT  +1)"));
```

Comment:
same here... wrap it in a `activeIf(.....clickable(<change>))` handler somewhere within the text..